### PR TITLE
Mebristo/60 download scripts

### PR DESCRIPTION
**Pull request by mebristo**
Add script to download files from AML Runs

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -83,5 +83,5 @@
     package_dir={"": "src"},
     include_package_data=True,
     install_requires=install_requires,
-    scripts=['src/health/azure/run_tensorboard.py']
+    scripts=['src/health/azure/run_tensorboard.py', 'src/health/azure/download_aml_run.py']
```
Comment (javier-alvarez) [inline setup.py:86]: Do we have docs for the cmd line tools? It would be good to add a link from main page

Comment (mebristo) [reply setup.py]: Have added to the docs

File: doc/examples.md [hunk 2]
```diff
@@ -31,3 +31,43 @@ hyperdrive_config = HyperDriveConfig(
...
+
+If `--experiment_name` is provided, the most recent Run from this experiment will be visualised.
+If `--latest_run_path` is provided, the script will expect to find a RunId in this file.
+Alternatively you can specify the Runs to visualise via  `--run_recovery_ids` or `--run_ids`.
+You can specify the location where TensorBoard logs will be stored, using the `--run_logs_dir` argument.
+
+If you choose to specify `--experiment_name`, you can also specify `--num_runs` to view and/or `--tags` to filter by.
+
+If your AML config path is not ROOT_DIR/config.json, you must also specify `--config_path`.
+<br><br>
```
Comment (dumbledad) [inline doc/examples.md:54]: Can we ditch HTML tags in markup, unless they're needed.

**Review by dumbledad — APPROVED** [review]

File: doc/examples.md (hunk 2 above)
Comment (mebristo) [reply doc/examples.md]: Done

**Pull request merged**